packages/manager/apps/web/client/app/hosting/cdn/flush/hosting-cdn-flush.routing.js

```diff
@@ -0,0 +1,27 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('app.hosting.dashboard.cdn.flush', {
+    url: '/flush',
+    layout: 'modal',
+    params: {
+      cdnProperties: null,
+      domain: null,
+      productId: null,
+    },
+    views: {
+      modal: {
+        component: 'hostingFlushCdnComponent',
+      },
+    },
+    resolve: {
+      cdnProperties: /* @ngInject */ ($transition$) =>
+        $transition$.params().cdnProperties,
+      domain: /* @ngInject */ ($transition$) => $transition$.params().domain,
+      goBack: /* @ngInject */ ($state) => () =>
+        $state.go('app.hosting.dashboard.multisite'),
+      onFlushSuccess: /* @ngInject */ ($rootScope) => () =>
+        $rootScope.$broadcast('hosting.cdn.flush.refresh'),
+      productId: /* @ngInject */ ($transition$) =>
```

Comment:
Not already declared in parent state ?

Reply:
No, the parent state `app.hosting.dashboard.multisite` is legacy code, which has no currently its `productId` defined in the routing

Reply:
https://github.com/ovh/manager/blob/master/packages/manager/apps/web/client/app/hosting/dashboard/hosting.routes.js#L171 the `app.hosting.dashboard` seems to

Reply:
Hmm ok I was looking for explicit `productId` ... You're right it is defined